src/app/CMakeLists.txt

```diff
@@ -145,10 +150,10 @@ if (STACKTRACE)
 endif()
 
 if (GUI)
-    set_target_properties(qbt_app PROPERTIES OUTPUT_NAME qbittorrent)
     target_link_libraries(qbt_app PRIVATE qbt_gui)
-else()
-    set_target_properties(qbt_app PROPERTIES OUTPUT_NAME qbittorrent-nox)
+    if ((CMAKE_SYSTEM_NAME STREQUAL "Windows") OR (CMAKE_SYSTEM_NAME STREQUAL "Darwin"))
```

Comment:
Please add a comment on top of this saying something like "NOTE: migrate to qt5_import_plugins() once support for Qt < 5.14 is dropped".

Reply:
Ok. Added.